biogeochem/EDPhysiologyMod.F90

```diff
@@ -828,7 +828,7 @@ subroutine trim_canopy( currentSite )
                 ! Check leaf cost against the yearly net uptake for that cohort leaf layer
                 if (currentCohort%year_net_uptake(z) < currentCohort%leaf_cost) then
                    ! Make sure the cohort trim fraction is great than the pft trim limit
-                   if (currentCohort%canopy_trim > EDPftvarcon_inst%trim_limit(ipft)) then
+                   if (currentCohort%canopy_trim > (EDPftvarcon_inst%trim_limit(ipft) + EDPftvarcon_inst%trim_inc(ipft))) then
```

Comment:
What predicated bumping up this check to include the increment?    Is it for the case in which `trim_limit` might be set less than the `trim_inc`?